pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -434,6 +439,9 @@ func (plugin *kubenetNetworkPlugin) SetUpPod(namespace string, name string, id k
 			// Not a hard error or warning
 			glog.V(4).Infof("Failed to clean up %s/%s after SetUpPod failure: %v", namespace, name, err)
 		}
+
+		// Perform garbage collection on allocated ips
+		plugin.ipamGarbageCollection()
```

Comment:
Hmm, is this going to hit our perf metrics? every pod create is stalled on docker ps -a right? does setupPod happen in parallel for each new pod?